wiki/Client/Beatmap_editor/AiMod/en.md

```diff
@@ -25,139 +25,143 @@ Development of AiMod usually lags behind the standards held by the mapping commu
-| <!-- Not sure how to test this one -->{0}'s dimensions must be {1}x{1} |  |  |
-| Your video's dimensions must not exceed 1024x768 for the 4:3 format. | *Self-explanatory* | Resize or replace the background video. |
-| Your video's dimensions must not exceed 1280x720 for the 16:9 format. | *Self-explanatory* | Resize or replace the background video. |
+| Message | Explanation | Solution | Notes |
+| :-- | :-- | :-- |  :-- |
+| Background image is larger than 2560x1440. | *Self-explanatory* | Resize or replace the background image. | |
+| This map may need an epilepsy warning, as it contains frequently toggled storyboards. | The storyboard contains elements that appear repeatedly in a very short time, possibly causing harm to uninitiated players who are prone to [seizures](https://en.wikipedia.org/wiki/Seizure). | Enable the `Display epilepsy warning (storyboard has quick strobing)` option in the Design tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup). | |
+| <!-- Not sure how to test this one -->{0}'s dimensions must be {1}x{1} | The dimension of the specified design element does not match its intended default. | Resize the design element in question to its proper resolution. | |
+| Your video's dimensions must not exceed 1024x768 for the 4:3 format. | *Self-explanatory* | Resize or replace the background video. | |
+| Your video's dimensions must not exceed 1280x720 for the 16:9 format. | *Self-explanatory* | Resize or replace the background video. | |
 
 #### Timing
 
-| Message | Explanation | Solution |
-| :-- | :-- | :-- |
-| This beatmap is over 6 minutes long. Consider shortening it if it's not a marathon-style map. |  |  |
-| Your beatmap is shorter than 45 seconds. Consider making it longer. |  |  |
-| Kiai time is toggled on for less than 15 seconds. | *Self-explanatory* | Extend the duration of Kiai time to longer than 15 seconds. |
+| Message | Explanation | Solution | Notes |
+| :-- | :-- | :-- |  :-- |
+| This beatmap is over 6 minutes long. Consider shortening it if it's not a marathon-style map. | *Self-explanatory* | Add more breaks into the beatmap or cut the song. | This warning can safely be ignored if the length of the map in question is intended. |
+| Your beatmap is shorter than 45 seconds. Consider making it longer. | *Self-explanatory* | Map out more parts of the song. | This warning can safely be ignored if the length of the map in question is intended. |
+| <!-- Potential removal? Not sure if this is broken, but AiMod doesn't seem to detect an issue when I use a 320Kbps audio file -->Audio bitrate is higher than 192kbps. Consider recompressing to CBR 192kbps or VBR ~1.0. | The beatmap's .mp3 file is encoded in a [bit rate](https://en.wikipedia.org/wiki/Bit_rate) that is higher than 192kbps, which goes above the [ranking criteria](/wiki/Ranking_Criteria)'s allowance. | Re-encode the .mp3 file so that its bit rate is between 128kbps and 192kbps (with 192kbps being the recommended option). | |
```

Comment:
abbreviations of units after a number should be preceded by a space

```suggestion
| <!-- Potential removal? Not sure if this is broken, but AiMod doesn't seem to detect an issue when I use a 320 kbps audio file -->Audio bitrate is higher than 192 kbps. Consider recompressing to CBR 192 kbps or VBR ~1.0. | The beatmap's `.mp3` file is encoded in a [bit rate](https://en.wikipedia.org/wiki/Bit_rate) that is higher than 192 kbps, which goes above the [ranking criteria](/wiki/Ranking_Criteria)'s allowance. | Re-encode the `.mp3` file so that its bit rate is between 128 kbps and 192 kbps (with 192 kbps being the recommended option). | |
```